concepticondata/conceptlists/Lundgren-2020-275.tsv

```diff
@@ -0,0 +1,276 @@
+Lundgren-2020-275-29	29	29	BEAT	1665	BEAT
+Lundgren-2020-275-30	30	30	BED	1663	BED
+Lundgren-2020-275-31	31	31	BELLY BUTTON	1838	NAVEL
+Lundgren-2020-275-32	32	32	BENT	297	CROOKED
+Lundgren-2020-275-33	33	33	BIG	1202	BIG
+Lundgren-2020-275-34	34	34	BIRD	937	BIRD
+Lundgren-2020-275-35	35	35	BITTER	887	BITTER
+Lundgren-2020-275-36	36	36	BLACK	163	BLACK
+Lundgren-2020-275-37	37	37	BLOOD	946	BLOOD
+Lundgren-2020-275-38	38	38	BOIL (TR.)	2492	BOIL (SOMETHING)
+Lundgren-2020-275-39	39	39	BONE	1394	BONE
+Lundgren-2020-275-40	40	40	BORROW	1823	BORROW
+Lundgren-2020-275-41	41	41	BRAIN	1583	BRAIN
+Lundgren-2020-275-42	42	42	BRANCH	1531	BRANCH
+Lundgren-2020-275-43	43	43	BREAST	1402	BREAST
+Lundgren-2020-275-44	44	44	BRING (TR.)	692	BRING
+Lundgren-2020-275-45	45	45	BROTHER (F. EGO)	560	BROTHER (OF WOMAN)
+Lundgren-2020-275-46	46	46	BURN	2102	BURN
+Lundgren-2020-275-47	47	47	BURST		
+Lundgren-2020-275-48	48	48	BURY (TR.)	1719	BURY
+Lundgren-2020-275-49	49	49	BUTTERFLY	1791	BUTTERFLY
+Lundgren-2020-275-50	50	50	CAIMAN	2438	CAIMAN
+Lundgren-2020-275-51	51	51	CALL (TR.)	1084	CALL
+Lundgren-2020-275-52	52	52	CANDELA~FIRE		
+Lundgren-2020-275-53	53	53	CANOE	1970	CANOE
+Lundgren-2020-275-54	54	54	CATFISH	3777	CATFISH
+Lundgren-2020-275-55	55	55	CHACRA	212	FIELD
+Lundgren-2020-275-56	56	56	CHAMELEON	1555	CHAMELEON
+Lundgren-2020-275-57	57	57	CHEST	1592	CHEST
+Lundgren-2020-275-58	58	58	CICADA	2353	CICADA
+Lundgren-2020-275-59	59	59	CLAY	1399	CLAY
+Lundgren-2020-275-60	60	60	CLOUD~FOG	3275	CLOUD OR FOG
+Lundgren-2020-275-61	61	61	COCKROACH	1575	COCKROACH
+Lundgren-2020-275-62	62	62	COLLARED PECCARY	435	COLLARED PECCARY
+Lundgren-2020-275-63	63	63	COME (INTR.)	1446	COME
+Lundgren-2020-275-64	64	64	COMPANION	1325	FRIEND
+Lundgren-2020-275-65	65	65	COOK	1100	COOK (SOMETHING)
+Lundgren-2020-275-66	66	66	COPULATE	549	COPULATE
+Lundgren-2020-275-67	67	67	CORN	506	MAIZE
+Lundgren-2020-275-68	68	68	COTTON	1850	COTTON
+Lundgren-2020-275-69	69	69	COVER (TR.)	761	COVER
+Lundgren-2020-275-70	70	70	CROSS	1367	CROSS
+Lundgren-2020-275-71	71	71	CRY (INTR.)	1839	CRY
+Lundgren-2020-275-72	72	72	CULTIVATE (TR.)	1541	CULTIVATE
+Lundgren-2020-275-73	73	73	CURE	499	CURE
+Lundgren-2020-275-74	74	74	CUT	1432	CUT
+Lundgren-2020-275-75	75	75	CUT HAIR	2155	CUT (WITH SCISSORS)
+Lundgren-2020-275-76	76	76	CUTBANK		
+Lundgren-2020-275-77	77	77	DANCE (INTR.)	1879	DANCE
+Lundgren-2020-275-78	78	78	DAUGHTER (M. EGO)		
+Lundgren-2020-275-79	79	79	DAWN	729	DAWN
+Lundgren-2020-275-80	80	80	DEEP	1593	DEEP
+Lundgren-2020-275-81	81	81	DEER	1936	DEER
+Lundgren-2020-275-82	82	82	DEFECATE (INTR.)	676	SHIT (DEFECATE)
+Lundgren-2020-275-83	83	83	DIG	1418	DIG
+Lundgren-2020-275-84	84	84	DOOR	1567	DOOR
+Lundgren-2020-275-85	85	85	DRIP (INTR.)		
+Lundgren-2020-275-86	86	86	DRUNK	1463	DRUNK
+Lundgren-2020-275-87	87	87	DRY (V.)		
+Lundgren-2020-275-88	88	88	DUST~SAND		
+Lundgren-2020-275-89	89	89	EAR	1247	EAR
+Lundgren-2020-275-90	90	90	EGG	744	EGG
+Lundgren-2020-275-91	91	91	ENCOUNTER		
```

Comment:
MEET I suppose.

Reply:
Unless it's also about encountering an object.

Reply:
Interesting observation, the Kokama lexeme under this concept actually mean 'respond, answer' (A Linguistic Analysis of Old Omagua Ecclesiastical Texts by Michael and O'Hagan (2016), Pequeno vocabulário Português-Tupi (Barbosa, 1970)), while the Omagua lexeme can mean both 'respond' and 'encounter' (the latter in the sense of experiencing something as an event, not literally as in 'meet a person'). So if we are to stick to Lundgren, I'd leave this one unmapped.